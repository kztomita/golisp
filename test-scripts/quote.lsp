(print (quote (1 2 a (+ 1 2))))
(print '(1 2 a (+ 1 2)))
