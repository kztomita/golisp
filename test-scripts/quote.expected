
(1 2 a (+ 1 2)) 
(1 2 a (+ 1 2)) 
